HIP/hip-820.md

```diff
@@ -71,38 +71,34 @@ When a dApp requires only the signature from the controller (wallet), it can use
 
-### hedera_signTransactionAndSend
-When a dApp needs the services of the controller to sign a transaction message and then submit it to the network, it can use the `hedera_signTransactionAndSend` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, appends it’s signature to the [`SignatureMap`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/basic_types.proto#L780) structure identified as the [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) property of the [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) repackages the message as appropriate for transmission to the Hedera network and sends it to the network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
+### hedera_signAndExecuteTransaction
+When a dApp needs the services of the controller to sign a transaction message and then submit it to the network, it can use the `hedera_signTransactionAndSend` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`TransactionList`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/sdk/transaction_list.proto) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, appends it’s signature to the [`SignatureMap`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/basic_types.proto#L780) structure identified as the [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) property of the [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) repackages the message as appropriate for transmission to the Hedera network and sends it to the network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
```

Comment:
`signAndSendTransaction` was chosen to more accurately express what the method does.  It does not wait for execution, it returns precheck values, which does not necessarily imply execution.  Also, it is good to use "Send" as to be distinct from previous efforts that may still be utilized in the WC system.  I do not believe this method name should be changed.

Reply:
@bugbytesinc understood and appreciate that opinion.

the cause for updating to "execute" vs. "send" is that it more closely aligns with the SDK terminology of execute.

```go
//Sign with client operator private key and submit the transaction to a Hedera network
txResponse, err := transaction.Execute(client)
```
```javascript
const txResponse = await transaction.execute(client);
```
```java
TransactionResponse txResponse = transaction.execute(client);
```

let's do a straw poll on this, please thumbs up which naming structure is preferred

Reply:
I understand both points of view. But there is another name that could be also considered: "call" (used in Signer)

Reply:
Great, thanks for the feedback @Volind 

Sounds like there's not a strong opinion either way. While I'd have a preference for what I've suggested, it's ultimately not up to me, and I definitely support not making this update. Just wanted to provide an alternative.

So as I tally:
Jason - strong opinion in favor of not accepting this change
Tyler & Volind - good with either option

We can keep this open for the rest of the day to allow for collecting of any other strong opinions